<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PsdWidget</class>
 <widget class="QWidget" name="PsdWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>598</height>
   </rect>
  </property>
  <widget class="QTreeView" name="treeView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>111</width>
     <height>241</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SelectionMode::ExtendedSelection</enum>
   </property>
  </widget>
  <widget class="QWidget" name="attributes" native="true">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>311</width>
     <height>373</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::StandardButton::Apply|QDialogButtonBox::StandardButton::Discard</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="type">
      <property name="title">
       <string>Type</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QRadioButton" name="typeEmbed">
         <property name="text">
          <string>&amp;Embed</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="embedWithTouch">
         <property name="text">
          <string>with &amp;Touch</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="typeMerge">
         <property name="text">
          <string>&amp;Merge to </string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="merge"/>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="typeCustom">
         <property name="text">
          <string>&amp;Custom</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="customEnabled">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="custom"/>
         </item>
         <item>
          <widget class="QComboBox" name="customBase"/>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QRadioButton" name="typeNative">
         <property name="text">
          <string>&amp;Native</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="nativeBase"/>
       </item>
       <item row="4" column="0">
        <widget class="QRadioButton" name="typeSkip">
         <property name="text">
          <string>&amp;Skip</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="properties">
      <property name="title">
       <string>Properties</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="visible">
         <property name="text">
          <string>&amp;Visible</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="color">
         <property name="text">
          <string>Co&amp;lor</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="position">
         <property name="text">
          <string>&amp;Position</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="text">
         <property name="text">
          <string>Te&amp;xt</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="size">
         <property name="text">
          <string>Si&amp;ze</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="image">
         <property name="text">
          <string>&amp;Image</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>18</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>60</y>
     <width>568</width>
     <height>205</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>false</bool>
   </property>
   <widget class="QPsdView" name="psdView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>125</width>
      <height>205</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPsdView</class>
   <extends>QWidget</extends>
   <header>qpsdview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
